### ISSUE TITLE
Bad Visual feedback results for Columns and Cells 

### ISSUE THREAD

**Issue by kmanojkkmr** [shahrukhqasim/TIES-2.0#39]
Hello @shahrukhqasim and Team,

I'm getting very bad results for Columns and Cells Prediction. But rows prediction is good. 
Could someone look into this and help me on this. 

I have modified "is_sampling_balanced = 0" from 1 to overcome "indices does not index into param shape" issue.
In the pdf's 
Blue rectangle indicates - Ground_Truth = 0 and Predicted = 0
Pink rectangle indicates - Ground_Truth = 1 and Predicted = 0
Orange rectangle indicates - The test cell which we are using for prediction of Cells/Columns/Rows

[02916_cells.pdf](https://github.com/shahrukhqasim/TIES-2.0/files/5217045/02916_cells.pdf)
[02916_cols.pdf](https://github.com/shahrukhqasim/TIES-2.0/files/5217046/02916_cols.pdf)

Thanks in advance.

**Comment by Sharathmk99** [comment]
Hi @kmanojkkmr What was the value of `samples_per_vertex`.
If your working on building table structure model, we can also collaborate and work together. Interested, please email me.

**Comment by zaocan666** [comment]
any solution yet？

**Comment by kmanojkkmr** [comment]
After completing around 5500 iterations, validation predictions are good. But the testing predictions are very bad.

**Comment by ReikoMin** [comment]
@kmanojkkmr I was wondering how do you get the program to work? Can you help me? thank you
Ubuntu 18.04
Python 3.6
tensorflow 1.13.1

**Comment by zhaozheninside** [comment]
@Sharathmk99
Could you please tell me the right value  of samples_per_vertex? thanks

**Comment by oysz2016** [comment]
I got similar results from my experiments，The results of the rows are much better than the results of the cols
is any solution yet？

**Comment by zaocan666** [comment]
> I got similar results from my experiments，The results of the rows are much better than the results of the cols
> is any solution yet？

problem solved by training more iterations. I trained for 30K iterations and the results are good

**Comment by kbrajwani** [comment]
hey have you tried to inference on new images?

**Comment by ivaylojelev** [comment]
Try setting `loss_alpha`, `loss_beta` and `loss_gamma` all to 1. The total loss that your model seeks to minimize is a weighted average of the column, row and cell loss, and these parameters are the weights. If you are using the default config, it is possible `loss_alpha` and `loss_gamma` are set to 0, so the column and cell loss are not actually minimizing. 

If the problem persists, you can set `loss_alpha` to an even higher value.
